tests/oss-fuzz/build.sh

```diff
@@ -19,71 +19,21 @@
 # https://github.com/google/oss-fuzz/blob/master/projects/libavif/build.sh
 # It builds the different fuzz targets.
 
-# To test changes to this file:
-# - make changes and commit to your REPO
-# - run:
-#     git clone --depth=1 git@github.com:google/oss-fuzz.git
-#     cd oss-fuzz
-# - modify projects/libavif/Dockerfile to point to your REPO
-# - run:
-#     python3 infra/helper.py build_image libavif
-#     # enter 'y' and wait for everything to be downloaded
-# - run:
-#     python3 infra/helper.py build_fuzzers --sanitizer address libavif
-#     # wait for the tests to be built
-# And then run the fuzzer locally, for example:
-#     python3 infra/helper.py run_fuzzer libavif \
-#     avif_fuzztest_enc_dec_incr@EncodeDecodeAvifFuzzTest.EncodeDecodeGridValid \
-#     --sanitizer address
-
-ln -s $SRC/fuzztest $SRC/libavif/ext/fuzztest
-
-# build dependencies
-cd ext && bash aom.cmd && bash dav1d.cmd && bash googletest.cmd && bash libjpeg.cmd && \
-      bash libsharpyuv.cmd && bash libyuv.cmd && bash zlibpng.cmd && cd ..
+# build dav1d
```

Comment:
This builds not only dav1d but also libyuv. It may be simpler to use the original comments "build dependencies".